### PR TITLE
CCMap ETL

### PR DESCRIPTION
**Pull request by m0nhawk**
Jira Ticket: [COV-170](https://occ-data.atlassian.net/browse/COV-170)

### New Features
* Added CCMap ETL

### CODE REVIEW

File: covid19-etl/etl/ccmap.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        print("Getting data from {}".format(url))
+        with closing(requests.get(url, stream=True)) as r:
+            f = (line.decode("utf-8") for line in r.iter_lines())
+            reader = csv.reader(f, delimiter=",", quotechar='"')
+
+            headers = next(reader)
+
+            if headers[0] == "404: Not Found":
+                print("  Unable to get file contents, received {}.".format(headers))
+                return
```
Comment (paulineribeyre) [inline covid19-etl/etl/ccmap.py:130]: should it error?

Comment (m0nhawk) [reply covid19-etl/etl/ccmap.py:130]: I have taken that from JHU: https://github.com/uc-cdis/covid19-tools/blob/master/covid19-etl/etl/jhu.py#L177-L179

Comment (paulineribeyre) [reply covid19-etl/etl/ccmap.py:130]: for JHU there was 1 link that didn't have data yet so it was expected

but here (and in other ETLs) i think it should error

Comment (paulineribeyre) [reply covid19-etl/etl/ccmap.py:130]: it should also error in JHU now actually

**Pull request merged**